src/ProjectTemplates/test/SpaTemplateTest/SpaTemplateTestBase.cs

```diff
@@ -83,7 +82,7 @@ public class SpaTemplateTestBase : BrowserTestBase
             Assert.True(0 == buildResult.ExitCode, ErrorMessages.GetFailedProcessMessage("build", Project, buildResult));
 
             // localdb is not installed on the CI machines, so skip it.
-            var shouldVisitFetchData = !useLocalDb;
+            var shouldVisitFetchData = !(useLocalDb && Project.IsCIEnvironment);
```

Comment:
shouldn't `&&` be `||`?

Reply:
No, we want this to run locally, but not on the CI

Reply:
I may be missing something but the expression currently means data will be fetched in CI builds when `!useLocalDb`.